README.md

```diff
@@ -42,6 +42,42 @@ job("foo") {
 }
 ```
 
+### Descriptive pipeline
```

Comment:
```suggestion
### Declarative pipeline
```

Jenkins documentation refers to it as "declarative pipeline", e.g., [here](https://www.jenkins.io/doc/book/pipeline/syntax/)